jobs/integr8ly/uninstallation-pipeline-qe-pony.yaml

```diff
@@ -0,0 +1,73 @@
+---
+
+- job:
+    name: uninstallation-pipeline-qe-pony
+    project-type: pipeline
+    description: "Uninstalls Integreatly remotely from QE Pony cluster by executing Ansible uninstallation playbook."
+    sandbox: false
+    concurrent: true
+    parameters:
+        - string:
+            name: REPOSITORY
+            default: https://github.com/integr8ly/installation.git
+            description: "Repository of the Integreatly installer"
+        - string:
+            name: BRANCH
+            default: 'master'
+            description: "Branch of the installer repository"
+        - string:
+            name: CLUSTER_URL
+            description: "URL of cluster for uninstallation."
+        - string:
+            name: MASTER_URLS
+            description: "Comma separated list of URLs for master nodes of target cluster to be used in Ansible inventory file"
+        - string:
+            name: OC_USER
+            description: "OpenShift user for QE cluster."
+        - string:
+            name: OC_PASSWORD
+            description: "OpenShift password for QE cluster."
+    dsl: |
+        timeout(60) {
+            node('cirhos_rhel7') {        
+                stage('Verify input') {
+                    if (!MASTER_URLS) {
+                        throw new hudson.AbortException('MASTER_URLS parameter is required!')
+                    }
+                }
+
+                stage('Clone the installer') {
+                    dir('installation') {
+                        git branch: BRANCH, url: REPOSITORY
+                    } // dir
+                } // stage
+
+                stage('Prepare environment') {
+                    dir('installation') {
+                        sh """
+                            cp ./evals/inventories/hosts.template ./evals/inventories/hosts
+                            sed -i 's/ansible_user=ec2-user/ansible_user=${ANSIBLE_USER}/g' ./evals/inventories/hosts
```

Comment:
I don't see ANSIBLE_USER parameter specified in the list of parameters

Reply:
Oh yeah good catch. THanks